tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -0,0 +1,829 @@
+    if (ifmode != mode) or (naming_mode != mode):
+        logger.info('Removing the created guest user')
+        duthost.user(name='guest', groups='sudo', state ='absent', shell='/bin/bash', remove='yes')
+        pytest.fail('Interface naming mode in .bashrc "{}", returned by show interfaces naming_mode "{}" does not the match the configured naming mode "{}"'.format(ifmode, naming_mode, mode))
+
+    yield dutHostGuest, mode, ifmode
+
+    logger.info('Removing the created guest user')
+    duthost.user(name='guest', groups='sudo', state ='absent', shell='/bin/bash', remove='yes')
+
+@pytest.fixture(scope='module')
+def sample_intf(setup):
+    """
+    Selects and returns the alias, name and native speed of the test interface
+
+    Args:
+        setup: Fixture defined in this module
+    Returns:
+        sample_intf: a dictionary containing the alias, name and native
+        speed of the test interface
+    """
+    interface = sorted(setup['up_ports'])[0]
+    interface_info = dict()
+    interface_info['default'] = interface
+    interface_info['alias'] = setup['port_name_map'][interface]
+    interface_info['native_speed'] = setup['port_speed'][interface_info['alias']]
+
+    return interface_info
+
+#############################################################
+######################## START OF TESTS #####################
+#############################################################
+
+# Tests to be run in all topologies
+
+class TestShowLLDP():
+
+    @pytest.fixture(scope="class")
+    def lldp_interfaces(self, setup):
+        """
+        Returns the alias and names of the lldp interfaces
+
+        Args:
+            setup: Fixture defined in this module
+        Returns:
+            lldp_interfaces: dictionary containing lists of aliases and
+            names of the lldp interfaces
+        """
+        minigraph_neighbors = setup['minigraph_facts']['minigraph_neighbors']
+        lldp_interfaces = dict()
+        lldp_interfaces['alias'] = list()
+        lldp_interfaces['interface'] = list()
+
+        for key, value in minigraph_neighbors.items():
+            if 'server' not in value['name'].lower():
+                lldp_interfaces['alias'].append(setup['port_name_map'][key])
+                lldp_interfaces['interface'].append(key)
+
+        if len(lldp_interfaces['alias']) == 0:
+            pytest.skip('No lldp interfaces found')
+
+        return lldp_interfaces
+
+    def test_show_lldp_table(self, setup, setup_config_mode, lldp_interfaces):
+        """
+        Checks whether 'show lldp table' lists the interface name as per
+        the configured naming mode
+        """
+        dutHostGuest, mode, ifmode = setup_config_mode
+        minigraph_neighbors = setup['minigraph_facts']['minigraph_neighbors']
+
+        lldp_table = dutHostGuest.shell('SONIC_CLI_IFACE_MODE={} show lldp table'.format(ifmode))['stdout']
+        logger.debug('lldp_table:\n{}'.format(lldp_table))
```

Comment:
This log message will only be output if logging level is DEBUG. When logging level is DEBUG, the previous call to the shell module would log the module execution result too. It looks like duplicated logs. Maybe you can consider raising the logging level to info here. Similar comments for other logger.debug calls later in the script.

Reply:
The logging level has been changed from DEBUG to INFO.